Dockerfile

```diff
@@ -15,6 +15,9 @@ RUN curl -fsSL "https://github.com/GoogleCloudPlatform/docker-credential-gcr/rel
     | tar xz --to-stdout ./docker-credential-gcr > /usr/local/bin/docker-credential-gcr \
     && chmod +x /usr/local/bin/docker-credential-gcr
 
+RUN wget https://github.com/roboll/helmfile/releases/download/v0.119.0/helmfile_linux_amd64 -O /bin/helmfile \
```

Comment:
Why use "wget" instead of just Dockerfile's "ADD"?

Reply:
@nethunter If I use "ADD" so I need to commands "ADD ${URL} /bin/helmfile" and "RUN chmod +x /bin/helmfile".

As well with the "ADD" command the layer will not be cached =\ and each build helmfile will be fetched.

Reply:
Fair enough, then why not use curl that's already installed? :-) Just wondering why do we need both curl and wget?